src/database/DataTypes/Location.js

```diff
@@ -6,12 +6,71 @@
 import Realm from 'realm';
 
 export class Location extends Realm.Object {
-  // TODO
-  // getBreaches(database) <- ('SensorLogs') filter this ID and isInBreach = true
-  // getBatches ?
+  // TODO remove data object
+  data = {
+    isInBreach: false,
+    totalStock: 16,
+    currentTemperature: '5.2',
+    temperatureExposure: { minTemperature: 2.5, maxTemperature: 10 },
+    temperaturePoints: {
+      maxLine: [
+        { date: 'Feb 23', temp: 5 },
+        { date: 'Feb 24', temp: 8 },
+        { date: 'Feb 25', temp: 8.5 },
+        { date: 'Feb 26', temp: 9 },
+        { date: 'Feb 27', temp: 10 },
+        { date: 'March 1', temp: 7 },
+        { date: 'March 2', temp: 5 },
+        { date: 'March 3', temp: 4 },
+        { date: 'March 4', temp: 5 },
+        { date: 'March 5', temp: 6 },
+      ],
+      minLine: [
+        { date: 'Feb 23', temp: 2.5 },
+        { date: 'Feb 24', temp: 5 },
+        { date: 'Feb 25', temp: 5.7 },
+        { date: 'Feb 26', temp: 7.5 },
+        { date: 'Feb 27', temp: 8 },
+        { date: 'March 1', temp: 6 },
+        { date: 'March 2', temp: 3.4 },
+        { date: 'March 3', temp: 3 },
+        { date: 'March 4', temp: 2.5 },
+        { date: 'March 5', temp: 4 },
+      ],
+      hazards: [{ date: 'Feb 27', temp: 10, onClick: () => console.log('CLICKED HAZARD 4') }],
```

Comment:
I think this `onClick` should be a prop of `VaccineChart`, from which props can be passed back. I think then, the hazard objects will need to have a `sensorLog` or `sensorLogID`. I don't see how we're going from clicking this object with no data, to opening a modal with a lot of information

Reply:
yeah for sure, hazard will have sensorLog, and i agree, onHazardPress can be just one prop of chart. Will refactor once aggregateLogs branch is merged.
This reminds me, need one more method that takes result from Wills aggregateLogs and result from extractBreaches and produces a breaches to render, results:
```
[{
timestamp: // must be the same value as the one on chart
temperature: // would come from sensorLog
sensorLogs: (breach)
}]
```

Reply:
I think this is this method here #886

Reply:
Ok, cheers I'll make a comment about params it's getting